modules/ossm-cert-manager-verify.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+
+:_content-type: PROCEDURE
+[id="ossm-verify-cert-manager-installation_{context}"]
+= Verify cert-manager installation
+
+.Procedure
+
+. Deploy the HTTP and `sleep` apps:
++
+[source, terminal]
+----
+oc new-project <project-name>
+oc apply -f https://raw.githubusercontent.com/maistra/istio/maistra-2.4/samples/httpbin/httpbin.yaml
+oc apply -f https://raw.githubusercontent.com/maistra/istio/maistra-2.4/samples/sleep/sleep.yaml
+----
+
+. Verify that `sleep` can access `httpbin`:
++
+[source, terminal]
+----
+oc exec "$(oc get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name})" -c sleep -n <project-name> -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "%{http_code}\n"
+----
++
+.Example output:
+[source, terminal]
+----
+200
+----
+
+. Check mTLS traffic from the ingress gateway to `httpbin`:
```

Comment:
```suggestion
. Check mTLS traffic from the ingress gateway to `httpbin`:
```
- literal's need a reference. What is `httpbin`? A component? An application? An object?